src/main/webapp/resources/js/pages/pipelines/pipeline.launch.js

```diff
@@ -129,9 +129,9 @@ function PipelineController(
    */
   vm.launchButtonTitle = function () {
     if (this.shouldDisableLaunch()) {
-      return window.PAGE.i18n.launchButtonTitleDisarmed;
+      return window.__i18n("workflow.launch.btn.title-disarmed")
     } else {
-      return window.PAGE.i18n.launchButtonTitleArmed;
+      return window.__i18n("workflow.launch.btn.title");
```

Comment:
Bunch of error in this page in the console:
![image](https://user-images.githubusercontent.com/8951708/70639843-add1b180-1c00-11ea-9d04-c8a285afd467.png)


Plus mousing over the launch button shows a non-internationalized text
![image](https://user-images.githubusercontent.com/8951708/70639984-dfe31380-1c00-11ea-9fcd-bc63e3337c60.png)

Reply:
Fixed in 54dbdccef29ff0c21eeb90b35ee6550847400fcb